packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.html

```diff
@@ -287,6 +287,10 @@
                 >
                     <thead>
                         <tr>
+                            <th class="oui-table__header">
+                                {{:: 'pci_projects_project_training_jobs_name' |
+                                translate}}
```

Comment:
You can use `data-translate='pci_projects_project_training_jobs_name'` directly on `th` element.

Reply:
done